srcpkgs/n30f/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'n30f'
+pkgname=n30f
+version=2.0
+revision=1
```

Comment:
add `build_style=gnu-makefile` below

Reply:
you know what, scratch that. the Makefile is broken for cross building. add the following in place of make:

    $(CC) $(CFLAGS) $(LDFLAGS) -o n30f n30f.c $(LDLIBS) -lcairo -lxcb -lxcb-render